src/routes/routeConfig.ts

```diff
@@ -53,6 +59,28 @@ export const PUBLIC_ROUTE_CONFIG: Record<string, IRoute> = {
     title: 'Register - BeeQuant',
     component: Register,
   },
+
+  [ROUTE_KEY.RESET_PASSWORD_INITIATION]: {
+    path: '/reset-password-initiation',
```

Comment:
since our url design is more Restful style, it's better to be like `/password/reset/initiate`